src/test/resources/reference.conf

```diff
@@ -74,6 +74,7 @@ elasticsearch {
 duos {
   baseConsentUrl = "http://localhost:8992"
   baseOntologyUrl = "http://localhost:8993"
+  dulvn = 3
```

Comment:
i think this also need to be 4

Reply:
This is really interesting. Since this is in a test resource, we should be considering whether we need to maintain tests for multiple versions. Unless someone can think of a problem that the current changes would cause, I propose that we don't worry about it right now but keep it in mind for the future.

Reply:
yes, that is correct. moran has said we do not yet need to worry about maintaining compatibility with previous versions.